modules/auth/auth-core/src/main/java/auth/core/module/SilhouetteContext.java

```diff
@@ -10,6 +10,7 @@
 import static java.lang.annotation.ElementType.PARAMETER;
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
+// TODO what is this for?
 @BindingAnnotation
```

Comment:
So I can tell Guice that my class depends on instance (execution context) marked by this attribute